accounts/forms.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, *args, **kwargs):
             if value is not None and value not in choices:
                 widget.choices.insert(1, [value, value])
 
+        if self.instance.get_ticket() is None:
+            self.fields['is_ukpa_member'].disabled = True
```

Comment:
Could you add a comment here?  I don't understand what setting `disabled` is for.